src/bin/ipa_bench/cmd.rs

```diff
@@ -124,6 +151,13 @@ impl Command {
     }
 }
 
+fn get_input(path: &Option<PathBuf>) -> Result<Box<dyn io::Read>, io::Error> {
```

Comment:
I believe @akoshelev mentioned we should not use `&Option` type. here's some [interesting info](https://stackoverflow.com/questions/16504643/what-is-the-overhead-of-rusts-option-type) on some compiler optimizations that can take place. that being said, any reason you couldn't just remove the `&` here?
```suggestion
fn get_input(path: Option<PathBuf>) -> Result<Box<dyn io::Read>, io::Error> {